src/Compilers/CSharp/Portable/Symbols/NullableContextFlags.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using Roslyn.Utilities;
+
+namespace Microsoft.CodeAnalysis.CSharp.Symbols
+{
+    /// <summary>
+    /// Used by symbol implementations to represent the byte value of [NullableContext].
```

Comment:
>[NullableContext] [](start = 70, length = 17)

Are we actually using these values in the attribute application?

Reply:
Yes, in `PENamedTypeSymbol` and `PEMethodSymbol`.

---
In reply to: [296408481](https://github.com/dotnet/roslyn/pull/36152#discussion_r296408481) [](ancestors = 296408481)